script/Arcueid/Day 5/05_00_ARC05_10B.txt

```diff
@@ -76,39 +76,37 @@ Have the %{i}Burial Agency%{/i} return back to the homeland as soon as possible.
 [sha:c99744f06be4c6e942f65fe84a437cad514dc981]{
 -- Page 762, Offset 10728.
 -- 「おう。ここでの仕事はだいたい終わった。
-"Right. Our work here is almost finished.
+"Aight, guess our job's just about done then.
 }
 [sha:7f4968b5f92a65140a0ff97691be11b86d768919]{
 -- Page 762, Offset 10729.
 -- 現場の後処理は誰にやらせてもいいんだが……
-It doesn't really matter who's in charge of cleaning up the scene, but...
+Don't really matter who we'll get to clean up the scene, but...
 }
 [sha:7be201a45fb3d07698af813d9e9913c09352dcd5]{
 -- Page 762, Offset 10730.
 -- おいアンドー！」
-Hey, Ando!"
+'EY, ANDOU!"
```

Comment:
I think at some point early in the project it was decided that it was gonna be "Ando" because that's how they spelled it in the mats.
Problem is it falls in conflict with Gouto, who is not spelled Goto. Similarly because at some part of the project it was decided like that. (Gouto and Goto have pretty conflicting translations).
The version with the long vowel is obviously closer to how'd you say it, Some commentators suggest using the long vowel indicator instead, so Gotō and Andō, and maybe what the Japanese attempt are imitations of this (but missing the crucial part). 

The tendency to ignore long vowels in Japanese romanization has taken many ill forms, but that it convinced the Japanese themselves that it was the way to go is highly unfortunate. (Tokyo, for example)